apis/stardust/types/index.d.ts

```diff
@@ -411,6 +412,12 @@ declare namespace stardust {
          */
         convertTo(newType: string, forceUpdate?: boolean, forcePatch?: boolean): Promise<object>;
 
+        /**
+         * Toggles the chart data view
+         * @param showDataView If included, turns the toggle into a one way-only operation. If true it will only toggle to the view data table.

```

Comment:
Could use a better description

Reply:
I think the wording `true it will only toggle to the view data table` is a bit misleading. You wont toggle to data view if it is already in data view. So maybe something like this
```suggestion
         * @param showDataView If included, forces the chart into the specified state. True will show data view, and false will show the original chart.
```

Reply:
Also, we don't really mention in the rest of the description that if you call the function again, it will toggle back to the original chart

Reply:
Yes that is a lot better, thanks!